mysql-schema/mysql-schema-user.md

```diff
@@ -63,8 +63,9 @@ DESC mysql.user;
 | Password_expired       | enum('N','Y')     | NO   |      | N                 |       |
 | Password_last_changed  | timestamp         | YES  |      | CURRENT_TIMESTAMP |       |
 | Password_lifetime      | smallint unsigned | YES  |      | NULL              |       |
+| Max_user_connections   | int unsigned      | NO   |      | 0                 |       |
```

Comment:
In MySQL this column is called `max_user_connections` (all lowercase) instead. Any reason for this difference?

Reply:
```
mysql-9.2.0> SHOW FIELDS FROM mysql.user LIKE 'max\_%';
+----------------------+--------------+------+-----+---------+-------+
| Field                | Type         | Null | Key | Default | Extra |
+----------------------+--------------+------+-----+---------+-------+
| max_questions        | int unsigned | NO   |     | 0       |       |
| max_updates          | int unsigned | NO   |     | 0       |       |
| max_connections      | int unsigned | NO   |     | 0       |       |
| max_user_connections | int unsigned | NO   |     | 0       |       |
+----------------------+--------------+------+-----+---------+-------+
4 rows in set (0.00 sec)
```

Reply:
I made a PR to fix it: https://github.com/pingcap/tidb/pull/59830
PTAL~